src/aura/disbursementsCreate/disbursementsCreate.cmp

```diff
@@ -1,128 +1,189 @@
-                            <dt class="slds-dl_inline__label slds-text-color_weak slds-truncate">Contact:</dt>
-                            <dd class="slds-dl_inline__detail slds-truncate">{!v.model.request.contactName}</dd>
-                            <dt class="slds-dl_inline__label slds-text-color_weak slds-truncate">Awarded Amount:</dt>
+                            <dt
+                                class="slds-dl_inline__label slds-text-color_weak slds-truncate"
+                            >
+                                {!v.model.request.fundingRequestLabels.Applying_Contact__c}:
+                            </dt>
+                            <dd class="slds-dl_inline__detail slds-truncate">
+                                {!v.model.request.contactName}
+                            </dd>
+                            <dt
+                                class="slds-dl_inline__label slds-text-color_weak slds-truncate"
+                            >
+                                {!v.model.request.fundingRequestLabels.Awarded_Amount__c}:
+                            </dt>
                             <dd class="slds-dl_inline__detail slds-truncate">
-                                <lightning:formattedNumber style="currency" value="{!v.model.request.awardedAmount}"/>
+                                <lightning:formattedNumber
+                                    style="currency"
+                                    value="{!v.model.request.awardedAmount}"
+                                />
                             </dd>
                         </dl>
                     </div>
 
-                    <div class="slds-p-horizontal_medium slds-size_1-of-2 slds-region_narrow">
+                    <div
+                        class="slds-p-horizontal_medium slds-size_1-of-2 slds-region_narrow"
+                    >
                         <dl class="slds-dl_inline slds-wrap">
-                            <dt class="slds-dl_inline__label slds-text-color_weak slds-truncate">Total Disbursed:</dt>
+                            <dt
+                                class="slds-dl_inline__label slds-text-color_weak slds-truncate"
+                            >
+                                {!v.model.request.fundingRequestLabels.Total_Disbursed__c}:
+                            </dt>
                             <dd class="slds-dl_inline__detail slds-truncate">
-                                <lightning:formattedNumber style="currency" value="{!v.model.request.totalDisbursed}"/>
+                                <lightning:formattedNumber
+                                    style="currency"
+                                    value="{!v.model.request.totalDisbursed}"
+                                />
                             </dd>
-                            <dt class="slds-dl_inline__label slds-text-color_weak slds-truncate">Amount Remaining:</dt>
+                            <dt
```

Comment:
The prettier print width limit broke up a few of the html elements is way that may not be obvious to community members. I wonder if we should add a note in the PR explaining why it formatted this way.

Reply:
I added links to prettier in the Pull Request body.   I can add more descriptions

Reply:
I already had a note in the Pull Request body.  Does it suffice?

![documented pretified disbersementsCreate component](https://user-images.githubusercontent.com/46537641/83800134-f2e91480-a674-11ea-90d9-db644a61ea27.png)